go-controller/pkg/node/node.go

```diff
@@ -368,7 +353,7 @@ func (n *OvnNode) Start(wg *sync.WaitGroup) error {
 	klog.Infof("Node %s ready for ovn initialization with subnet %s", n.name, util.JoinIPNets(subnets, ","))
 
 	if config.OvnKubeNode.Mode != types.NodeModeSmartNICHost {
-		if _, err = isOVNControllerReady(n.name); err != nil {
+		if err := wait.PollImmediate(500*time.Millisecond, 300*time.Second, n.isOVNControllerReady); err != nil {
```

Comment:
I agree that it's weird to be calling this twice. You should figure out if this one can be removed.

Reply:
I didn't find a reason, why we should keep it there, so I removed it (at the beginning I simply wasn't brave enough ^^)